src/mlpack/core.hpp

```diff
@@ -21,13 +21,7 @@
  * bindings to other languages.  It is meant to be a machine learning analog to
  * LAPACK, and aims to implement a wide array of machine learning methods and
  * function as a "swiss army knife" for machine learning researchers.  The
- * mlpack development website can be found at http://mlpack.org.
- *
- * mlpack uses the Armadillo C++ matrix library (http://arma.sourceforge.net)
- * for general matrix, vector, and linear algebra support.  mlpack also uses the
- * program_options, math_c99, and unit_test_framework components of the Boost
- * library, and optionally uses libbfd and libdl to give backtraces when
- * compiled with debugging symbols on some platforms.
+ * mlpack website can be found at http://mlpack.org.
```

Comment:
Maybe it's time to use https as a default?

Reply:
@zoq Does MlPack website support HTTPS?

Reply:
Agreed, I hadn't noticed that.  Fixed in bf5506e.